src/library/dao/trackdao.cpp

```diff
@@ -1381,7 +1381,7 @@ TrackPointer TrackDAO::getTrackById(TrackId trackId) const {
         QString columnsStr;
         int columnsSize = 0;
         for (int i = 0; i < columnsCount; ++i) {
-            columnsSize += qstrlen(columns[i].name) + 1;
+            columnsSize += static_cast<int>(qstrlen(columns[i].name)) + 1;
         }
         columnsStr.reserve(columnsSize);
```

Comment:
I would've preferred changing `columnsSize` to `std::size_t`/`qsizetype` instead.

Reply:
Yes that world be nice. However QT provides no common ground with qsizetype for QString.reserve(). So using `int` is the compatible solution.

Reply:
What do you mean, "no common ground"? `QString.reserve()` takes a `qsizetype` on Qt6...

Reply:
Shouldn't `auto` use `int` on Qt5 and `size_t` on Qt6?

Reply:
We need a cast anyway, because Qt6 uses std::ptrdifsize for reserve()